examples/chain_client/49_ChainStream.py

```diff
@@ -0,0 +1,62 @@
+import asyncio
+
+from google.protobuf import json_format
+
+from pyinjective.async_client import AsyncClient
+from pyinjective.composer import Composer
+from pyinjective.core.network import Network
+
+
+async def main() -> None:
+    network = Network.devnet()
```

Comment:
The rest of the examples point to testnet, for consistency it's best we use testnet here too.

Reply:
Will change this